### DISCUSSION TITLE
Return code from `adb install`

### DISCUSSION THREAD

**Discussion by pojungc** [yume-chan/ya-webadb#532] [General]
Hi!
Is there a way to see if the adb install was successful or there was an error? When I install an app through the CLI I get the a failure because of signature mismatch, however on the demo with the same `.apk`,  it indicates that the instal was completed with `Install finished in 16399ms at 8.36MB/s` with no indication that there was a failure. 

I think I can just do some polling if the version i'm installing has been updated on the device is the correct one, but i'd like to avoid this. 

Thank you :)

**Comment by yume-chan** [comment]
Which Android version your device is? I think `adb install` doesn't have exit code on Android <7 and >12 either.

**Reply by pojungc** [reply]
Ah, It is Android 10, so would the answer be that there is no way to know with the current implementation?
